pkg/destroy/gcp/backendservice.go

```diff
@@ -2,69 +2,100 @@ package gcp
 
 import (
 	"context"
+	"fmt"
 
-	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"google.golang.org/api/compute/v1"
 	"google.golang.org/api/googleapi"
 
 	"github.com/openshift/installer/pkg/types/gcp"
 )
 
-func (o *ClusterUninstaller) listBackendServices(ctx context.Context) ([]cloudResource, error) {
-	return o.listBackendServicesWithFilter(ctx, "items(name),nextPageToken", o.clusterIDFilter(), nil)
+func (o *ClusterUninstaller) listBackendServices(ctx context.Context, scope resourceScope) ([]cloudResource, error) {
+	return o.listBackendServicesWithFilter(ctx, "items(name),nextPageToken", o.clusterIDFilter(), nil, scope)
+}
+
+func createBackendResources(filterFunc func(*compute.BackendService) bool, list *compute.BackendServiceList) []cloudResource {
+	result := []cloudResource{}
+
+	for _, item := range list.Items {
+		if filterFunc == nil || filterFunc(item) {
+			logrus.Debugf("Found backend service: %s", item.Name)
+			result = append(result, cloudResource{
+				key:      item.Name,
+				name:     item.Name,
+				typeName: "backendservice",
+				quota: []gcp.QuotaUsage{{
+					Metric: &gcp.Metric{
+						Service: gcp.ServiceComputeEngineAPI,
+						Limit:   "backend_services",
+					},
+					Amount: 1,
+				}},
+			})
+		}
+	}
+
+	return result
 }
 
 // listBackendServicesWithFilter lists backend services in the project that satisfy the filter criteria.
 // The fields parameter specifies which fields should be returned in the result, the filter string contains
 // a filter string passed to the API to filter results. The filterFunc is a client-side filtering function
 // that determines whether a particular result should be returned or not.
-func (o *ClusterUninstaller) listBackendServicesWithFilter(ctx context.Context, fields string, filter string, filterFunc func(*compute.BackendService) bool) ([]cloudResource, error) {
-	o.Logger.Debugf("Listing backend services")
+func (o *ClusterUninstaller) listBackendServicesWithFilter(ctx context.Context, fields string, filter string, filterFunc func(*compute.BackendService) bool, scope resourceScope) ([]cloudResource, error) {
+	o.Logger.Debugf("Listing %s backend services", scope)
 	ctx, cancel := context.WithTimeout(ctx, defaultTimeout)
 	defer cancel()
 	result := []cloudResource{}
+
+	if scope == gcpGlobalResource {
+		req := o.computeSvc.BackendServices.List(o.ProjectID).Fields(googleapi.Field(fields))
+		if len(filter) > 0 {
+			req = req.Filter(filter)
+		}
+		err := req.Pages(ctx, func(list *compute.BackendServiceList) error {
+			result = append(result, createBackendResources(filterFunc, list)...)
+			return nil
+		})
+		if err != nil {
+			return nil, fmt.Errorf("failed to list global backend services: %w", err)
+		}
+		return result, err
```

Comment:
```suggestion
		return result, nil
```
to be consistent with the other similar changes.